libs/coin-modules/coin-aptos/src/bridge/estimateMaxSpendable.ts

```diff
@@ -29,7 +29,7 @@ const estimateMaxSpendable = async ({
     gasUnitPrice = BigNumber(estimate.gasUnitPrice);
   }
 
-  return getMaxSendBalance(mainAccount.spendableBalance, maxGasAmount, gasUnitPrice);
+  return getMaxSendBalance(maxGasAmount, gasUnitPrice, mainAccount, transaction);
```

Comment:
We edit this method and it doesn't require test update ?

Reply:
Here: https://github.com/LedgerHQ/ledger-live/pull/9451/files/a47e1f52092a2a955310260e596eff1a5e69562d